.buildkite/pipeline_mflowgen.yml

```diff
@@ -1,230 +1,57 @@
 steps:
 
-########################################################################
+##############################################################################
+# INDIVIDUAL TILE RUNS
+
 - label: 'MemCore synth 17m'
   agents: { jobsize: "hours" }
   commands:
   - test=mflowgen/test/test_module.sh
   - $$test full_chip tile_array Tile_MemCore --steps synthesis --debug
+- wait: { continue_on_failure: true } # One step at a time + continue on failure
```

Comment:
I thought you wanted it to stop on errors? Why is this true?

Reply:
> I thought you wanted it to stop on errors? Why is this true?

This "continue on failure" applies to the top-level jobs, not the individual steps between jobs. E.g. if Memcore is doing synthesis-place-route-lvs, and it fails in synthesis, it now halts immediately (as opposed to before, when it would go ahead and try to finish the run). But whether the MemTile test fails or succeeds, we still want to run the PE tile test etc.